src/banking_stage.rs

```diff
@@ -80,26 +92,33 @@ impl BankingStage {
             }).collect()
     }
 
+    fn tick(poh_service: &PohService, entry_sender: &Sender<Vec<Entry>>) -> Result<()> {
```

Comment:
Can you use EntrySender here? https://github.com/solana-labs/solana/blob/master/src/entry.rs#L16

Reply:
sure, didn't know about the type

it doesn't conflict with the "one file one style" rule?